sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/ContainerAsyncClient.java

```diff
@@ -1334,6 +1334,24 @@ public String generateSAS(String identifier, ContainerSASPermission permissions,
         return blobServiceSasQueryParameters.encode();
     }
 
+    /**
+     * Get the container name.
+     *
+     * <p><strong>Code Samples</strong></p>
+     *
+     * {@codesnippet com.azure.storage.blob.ContainerAsyncClient.getName}
+     *
+     * @return The name of container.
+     */
+    public String getName() {
+        try {
+            return URLParser.parse(new URL(this.azureBlobStorage.getUrl())).getBlobName();
```

Comment:
This is using `getBlobName` does that make sense?

Reply:
Rename to blobName